QuizApp/Custom classes/CustomTextField.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CustomTextField.swift
+//  QuizApp
+//
+//  Created by five on 28.05.2021..
+//
+
+import UIKit
+
+class CustomTextField: UITextField, UITextFieldDelegate {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+    
+    private func commonInit() {
+        textColor = .white
+        backgroundColor = .init(white: 1, alpha: 0.3)
```

Comment:
Using `UIColor(white: 1, alpha: 0.3)` would improve readability over `.init(...)`. And generally, we tend to use the class name as an init call rather than explicitly using `.init` (with or without the class name in front)